docker/Dockerfile

```diff
@@ -22,17 +22,39 @@ RUN apt-get -y update && apt-get clean
 RUN apt-get -y install  xvfb xserver-xephyr libxml2-dev libxslt-dev libssl-dev zlib1g-dev gcc python-all-dev \
 postgresql-server-dev-all postgresql-client postgresql-client-common \
 postgresql libcurl4-openssl-dev proxychains tor ca-certificates libpq-dev  \
-libxslt1-dev libldap2-dev libsasl2-dev libffi-dev net-tools lsof locales-all \
-libyaml-dev python3.9 python3.9-dev build-essential python3.9-venv python3-lxml python3-impacket golang-go python3-pip
+libxslt1-dev libldap2-dev libsasl2-dev libffi-dev net-tools lsof locales-all 
+#libyaml-dev python3.9 python3.9-dev build-essential python3.9-venv python3-lxml python3-impacket golang-go python3-pip
+# RUN apt-get install -y make build-essential libssl-dev git zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev
+# RUN curl https://pyenv.run | bash
+# RUN echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.bashrc
+# RUN echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.bashrc
+# RUN echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init --path)"\nfi' >> ~/.bashrc
+# RUN source ~/.bashrc
+# RUN exec $SHELL
+RUN apt-get install -y --no-install-recommends make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
+
+ENV HOME="/root"
+WORKDIR ${HOME}
+RUN apt-get install -y git
+RUN git clone --depth=1 https://github.com/pyenv/pyenv.git .pyenv
+ENV PYENV_ROOT="${HOME}/.pyenv"
+ENV PATH="${PYENV_ROOT}/shims:${PYENV_ROOT}/bin:${PATH}"
+
+ENV PYTHON_VERSION=3.9
+RUN pyenv install ${PYTHON_VERSION}
+RUN pyenv global ${PYTHON_VERSION}
+RUN pyenv --version
+RUN python --version
+
 
 # Set locale
 ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
 
-# Install optional tools (LBD, arachni, gnutls-bin, o-saft and metagoofil)
-RUN apt-get -y install lbd theharvester tlssled nikto dnsrecon nmap whatweb skipfish dirbuster \
-metasploit-framework wpscan wapiti hydra metagoofil o-saft amass
+# Install optional tools (LBD, arachni, gnutls-bin, o-saft and metagoofil) (not available on ubuntu20)
```

Comment:
without these tools, some of OWTF functionality will not be available - how can we use the same tools in Ubuntu, any ideas?

Reply:
we changed to kali, so its working

Reply:
yeah they can be installed as we are again using kali.
But getting this error at the end of docker build
![image](https://github.com/owtf/owtf/assets/54772821/9613812f-9db9-4613-87a6-35bdbd818e0b)

Reply:
I think that's related to your machine running out of memory while building the image :) https://github.com/docker/cli/issues/2971